### PR TITLE
Use the default Button theme

### PR DESCRIPTION
**Pull request by mattleibow**
### Description of Change ###

Try to not use the ancient themes.

Default theme: https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/CommonStyles/Button_themeresources.xaml

### CODE REVIEW

**Comment by jsuarezruiz** [comment]
And also implements CornerRadius property, right?

File: src/Core/src/Handlers/Button/ButtonHandler.iOS.cs [hunk 1]
```diff
@@ -44,6 +44,11 @@ void SetupDefaults(UIButton nativeView)
 			ButtonTextColorDefaultDisabled = nativeView.TitleColor(UIControlState.Disabled);
 		}
 
+		[PortHandler]
```
Comment (jsuarezruiz) [inline src/Core/src/Handlers/Button/ButtonHandler.iOS.cs:47]: [MissingMapper]

File: src/Core/src/Handlers/Button/ButtonHandler.Android.cs [hunk 2]
```diff
@@ -64,6 +64,11 @@ public static void MapBackground(ButtonHandler handler, IButton button)
 			handler.NativeView?.UpdateBackground(button, DefaultBackground);
 		}
 
+		[PortHandler]
```
Comment (jsuarezruiz) [inline src/Core/src/Handlers/Button/ButtonHandler.Android.cs:67]: [MissingMapper]

**Review by jsuarezruiz — COMMENTED** [review]
Could you include the `[PortHandler] `attribute in the UWP ButtonRenderer.

Apart some small comments, looks nice and works as expected:
Captura de pantalla 2021-07-16 a las 11 53 58

**Comment by Redth** [comment]
Superseded by win11 styles

**Pull request closed without merging**